apps/rush-lib/src/start.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { EOL } from 'os';
+import * as colors from 'colors';
+
+import rushVersion from './rushVersion';
+import RushCommandLineParser from './cli/actions/RushCommandLineParser';
+
+// tslint:disable-next-line:export-name
+export function executeCli(wrapperVersion: string, isManaged: boolean = false): void {
```

Comment:
>wrapperVersion [](start = 27, length = 14)

"wrapper" seems like the wrong term.  I think I would just call this "rushVersion" or "callerVersion" (or maybe "launcherVersion"?)